meteor/override-fastclick.js

```diff
@@ -0,0 +1,6 @@
+// disable fastclick package in Meteor if included
```

Comment:
@Urigo is there any way we can avoid this? The PR looks good, just we're not too keen on this.